client/src/Features/PublishAssignment/PublishSuccessMessageBar.tsx

```diff
@@ -58,13 +65,16 @@ const PublishSuccessMessageBarInner = ({
   return null;
 };
 
-const publishSuccessMessageBarStyles = ({ theme: _theme }: IThemeOnlyProps): PublishSuccessMessageBarStyles => ({
+const publishSuccessMessageBarStyles = ({ theme }: IThemeOnlyProps): PublishSuccessMessageBarStyles => ({
   root: [
-    {
+    mergeStyles(AnimationClassNames.fadeIn200, {
+      height: theme.spacing.l2,
       width: 'auto',
-      marginLeft: 'auto'
-    }
+      marginLeft: '32px',
+      marginRight: '32px',
```

Comment:
can we write these in `calc` format as well and not hardcode exact values in pixels wherever possible please